jack-sys/build.rs

```diff
@@ -1,3 +1,14 @@
+use bitflags::bitflags;
+
+use std::io::Write;
+
+bitflags! {
+    struct FunctionFlags: u8 {
+        const NONE = 0b00000000;
+        const WEAK = 0b00000001;
+    }
+}
+
 fn main() {
     println!("cargo:rerun-if-env-changed=RUST_JACK_DLOPEN");
     let dlopen = std::env::var("RUST_JACK_DLOPEN").is_ok();
```

Comment:
This doesn't do anything without the `dlopen` feature.

Reply:
Ack.

TODO(Fix). Possibly require this for just linux.